### PR TITLE
Added some style to the project

### PR DESCRIPTION
**Pull request by SaeqaSultani**
- Now you should improve the look and feel of the application, adding the styles of your choice.
- Please keep the general layout of the wireframe, this is the only mandatory requirement.
- can use plain CSS or any CSS framework, it's up to you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
 this is a webpack project that the purpose of this project is to understand how use callbacks and promises,
 learn how to use proper ES6 syntax,use ES6 modules to write modular JavaScript and use webpack to bundle JavaScript.
+
+![MyImage](./src/images/screenshot.png)
+
 # Built With
 - HTML
 - CSS
 - JavaScripth
 - Webpack
```
Comment (AmaduKamara) [inline README.md:12]: - [ ] Kindly add instructions below this one in your README on how someone will be able to start the app. Take note of the following:
- Cloning the repo
- cd into the directory
- installing the npm packages (`npm install`)
- And finally starting the app. (that will be `npm start`) 

Please add these instructions above below the built-with technologies so one will know to access and start your app. 👍

File: src/index.css [hunk 2]
```diff
@@ -91,25 +112,35 @@ body {
...
-  height: 55vh;
+  height: 450px;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding-top: 40px;
+  margin-bottom: 20px;
+  background-color: rgb(241, 172, 178);
 }
```
Comment (AmaduKamara) [inline src/index.css]: - [ ] Kindly app padding-top and bottom of this wrapper to have the first and last player's name not hidden as in the screenshot below:

image

- [ ] I tried this one and things were fine. `padding: 20px 10px;`

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @SaeqaSultani,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css [hunk 3]
```diff
@@ -91,25 +112,36 @@ body {
...
+  height: 50vh;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding-top: 40px;
+  margin-bottom: 20px;
+  background-color: rgb(241, 172, 178);
+  padding-top: 20px;
 }
```
Comment (roshan-bajgain) [inline src/index.css]: - [ ] I would suggest you please implement the changes requested by the previous reviewer. When I add the name and score then the first name and score are hidden. please fix it
### Your design
ssss

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @SaeqaSultani, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @SaeqaSultani,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**